app/views/api/v2/authorizable.json.rabl

```diff
@@ -0,0 +1,7 @@
+object @resource
+
+if @object.try(:class).try(:ancestors).to_a.include?(Authorizable)
```

Comment:
I'm wondering if this should be guarded by a special parameter, something like [thin](https://github.com/theforeman/foreman/blob/d5f2f1de95e4cc665705c5047ccc89660f0bb92d/app/controllers/api/v2/hosts_controller.rb#L35-L52). Should we introduce this in a separate PR? That might make reviewing easier. Does this have tests?